README.md

```diff
@@ -407,13 +407,13 @@ You can use these to imitate some of the component's behaviours.
 
 | Name | Arguments | Description |
 |------|-----------|-------------|
-|`onInputClick`| HTML click event | Fires whenever the input is being clicked. |
+|`showSuggestions`| HTML click event | Fires whenever the input is being clicked. Shows suggestion list (gets them from the source if none is present). Used to be named `onInputClick`. |
```

Comment:
I think it is better to keep the old name and set a lable `will be renamed` or something. And set `showSuggestions` as an alias